app/src/main/java/org/openobservatory/ooniprobe/model/NetworkMeasurement.java

```diff
@@ -116,7 +129,7 @@ public static int getTestImageBig(String name) {
                 return R.drawable.http_invalid_request_line_big;
             case OONITests.HTTP_HEADER_FIELD_MANIPULATION:
                 return R.drawable.http_header_field_manipulation_big;
-            case OONITests.NDT_TEST:
+            case OONITests.NDT:
```

Comment:
Here, for example, why are you `s/NDT_TEST/NDT/` and not adding another case?

Reply:
I don't need the long description. Only test name is stored in old versions

Reply:
Thanks!